optimum/onnxruntime/modeling_ort.py

```diff
@@ -1317,11 +1388,14 @@ class ORTModelForMultipleChoice(ORTModel):
     )
     def forward(
         self,
-        input_ids: Optional[torch.Tensor] = None,
+        input_ids: Optional[Union[torch.Tensor, np.ndarray]] = None,
         attention_mask: Optional[torch.Tensor] = None,
         token_type_ids: Optional[torch.Tensor] = None,
```

Comment:
These cannot be `np.ndarray`?

Reply:
Forgot this type hint, thanks for noticing!